File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-the-controls-on-a-ribbon-at-run-time_2.cs
        //Declare the Northwind dataset.
        Northwind40DataSet nwDataSet = new Northwind40DataSet();

        //Declare the data tables.

        Northwind40DataSet.CustomersDataTable customerTable;
        Northwind40DataSet.OrdersDataTable orderTable;
        Northwind40DataSet.Order_DetailsDataTable orderDetailsTable;
        Northwind40DataSet.ProductsDataTable productsTable;

        //Declare the data table adapters for each table.

        CustomersTableAdapter customerTableAdapter = new CustomersTableAdapter();
        OrdersTableAdapter ordersTableAdapter = new OrdersTableAdapter();
        Order_DetailsTableAdapter detailsTableAdapter = new Order_DetailsTableAdapter();
        ProductsTableAdapter productsTableAdapter = new ProductsTableAdapter();

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-the-controls-on-a-ribbon-at-run-time_2.vb
    'Declare the Northwind data set.

    Dim nwDataSet As Northwind40DataSet = New Northwind40DataSet()

    'Declare the data tables.

    Dim customerTable As Northwind40DataSet.CustomersDataTable
    Dim orderTable As Northwind40DataSet.OrdersDataTable
    Dim orderDetailsTable As Northwind40DataSet.Order_DetailsDataTable
    Dim productsTable As Northwind40DataSet.ProductsDataTable

    'Declare the data table adapters for each table.

    Dim customersTableAdapter As CustomersTableAdapter = New CustomersTableAdapter()
    Dim ordersTableAdapter As OrdersTableAdapter = New OrdersTableAdapter()
    Dim detailsTableAdapter As Order_DetailsTableAdapter = New Order_DetailsTableAdapter()
    Dim productsTableAdapter As ProductsTableAdapter = New ProductsTableAdapter()
